dhcp_server/DOCS.md

```diff
@@ -132,6 +137,31 @@ The MAC address of the client device.
 
 The IP address you want the DHCP server to assign.
 
+### Option: `static_routes` (optional)
+
+This option defines settings for DHCP option 121 (Classless Static Route).
+It allows you to inject static network route with DHCP configuration.
+
+By default, non are configured.
+
+#### Option: `static_routes.subnet`
+
+The network schema/subnet (`networks.subnet`) for which the rule will apply.
+
```

Comment:
With this config a user would do something like this:
```yaml
networks:
  - subnet: 192.168.1.0
    ...
static_routes:
  - subnet: 192.168.1.0
    network: 192.168.2.0
    mask: 24
    gateway: 192.168.1.3
```
Instead of making them repeat `subnet`, lets just embed this in the network items. Like this:
```yaml
networks:
  - subnet: 192.168.1.0
    ...
    static_routes:
      - network: 192.168.2.0
        mask: 24
        gateway: 192.168.1.3
```